### PR TITLE
New article "WebView2 external resources"

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered article for review:**
[Resources, talks, videos, and blogs about WebView2](https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/resources?branch=pr-en-us-1380)
**Rendered article from other PR:**
[Resources, talks, videos, and blogs about DevTools](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/resources?branch=pr-en-us-1386)

- Change title of article from "Resources, talks, videos, and blogs about Microsoft Edge WebView2" to "WebView2 external resources"?
- Add more links?

### CODE REVIEW

File: microsoft-edge/webview2/resources.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+description: Resources, talks, videos, and blogs about Microsoft Edge WebView2
+title: Resources, talks, videos, and blogs about Microsoft Edge WebView2
```
Comment (mikehoffms) [inline microsoft-edge/webview2/resources.md:3]:
```suggestion
title: Resources, talks, videos, and blogs about WebView2
```

File: microsoft-edge/webview2/resources.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 08/09/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+# Resources, talks, videos, and blogs about Microsoft Edge WebView2
```
Comment (mikehoffms) [inline microsoft-edge/webview2/resources.md:12]:
```suggestion
# Resources, talks, videos, and blogs about WebView2
```

File: microsoft-edge/webview2/resources.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+| [WebView2: Bringing the best of the web to your native apps](https://www.youtube.com/watch?v=-ri7TmPeqLc) | From the Microsoft Ignite 2020 conference. | YouTube channel: Microsoft Edge.  Length: 13:36.  Uploaded September 22, 2020. |
+| [Announcing Microsoft Edge WebView2 for Office Add-Ins Platform](https://www.youtube.com/watch?v=D73D-EB89Fs) | How we are evolving the web viewer control used by the Office add-ins platform for add-ins running on Windows.  A talk from Office Add-Ins. | YouTube channel: Microsoft 365 Developer.  Length: 5:07.  Uploaded August 12, 2020. |
+| [WebView2: Bringing the best of the web to your native apps \| BDL105](https://www.youtube.com/watch?v=P0K3DWp05QQ) | From Microsoft Build 2020. | YouTube channel: Microsoft Developer.  Length: 14:37.  Uploaded May 24, 2020. |
+  
+About the Ignite conference at which some of these videos were presented: [Microsoft Ignite 2020](https://news.microsoft.com/ignite2020/): Microsoft's annual gathering of technology leaders and practitioners.
```
Comment (vchapel) [inline microsoft-edge/webview2/resources.md:28]: Only one comment on this page ... two colons in one sentence. I don't know if there's any way around this, but if feels awkward.

Comment (mikehoffms) [reply microsoft-edge/webview2/resources.md:27]: Way around this: delete and streamline to concentrate attention on the video title.

**Comment by mikehoffms** [comment]
@liminzhu @jasonstephen15 This PR is out for review.  Do you have any other items to add?  Please review & Approve this PR.

**Review by jasonstephen15 — APPROVED** [review]

**Pull request closed without merging**